pkg/objstore/inmem.go

```diff
@@ -164,6 +169,10 @@ func (b *InMemBucket) Upload(_ context.Context, name string, r io.Reader) error
 		return err
 	}
 	b.objects[name] = body
+	b.attrs[name] = ObjectAttributes{
+		Size:         int64(len(body)),
+		LastModified: time.Now(),
```

Comment:
I guess some mocked `now func() time.Time` would be nice (:

Reply:
But would that be used anywhere? Isn't it a case YAGNI? :P

Reply:
it will be used here. (: